js-fetch_star-wars-console/js/index.js

```diff
@@ -1,7 +1,20 @@
 console.clear();
 
-const url = "https://swapi.dev/api/people";
+const url = "https://swapi.dev/api/peoplepppp";
 
-function fetchData() {}
+async function fetchData() {
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+
+    if (response.ok) {
+      console.log(data.results[2].eye_color);
+    } else {
+      console.error("Moin");
```

Comment:
If you prefer, you could separate the parsing logic from the fetching logic to isolate responsibilities

Reply:
cool thank you!